admin/includes/footer.php

```diff
@@ -0,0 +1,38 @@
+
+<!--admin footer start-->
+ < <footer id="admin_footer" class="py-4 bg-light">
```

Comment:
There is an extra < character here. Please remove :)

Reply:
removed